### PR TITLE
Main notebook

### PR DESCRIPTION
**Pull request by raphaelattias**
- rewrote the trainloader with native torchvision api for data augmentation. Added arg for transform and noARA so we don't have redondant code
- train can now track the loss on a validation set passsed as argument, the output is the history
- added saturation, rotation, brightness, flip etc.

### CODE REVIEW

**Comment by riccardocadei** [comment]
well done
